test/files/normalise-input.spec.js

```diff
@@ -0,0 +1,154 @@
+'use strict'
+
+/* eslint-env mocha */
+const chai = require('chai')
+const dirtyChai = require('dirty-chai')
+const normalise = require('../../src/files/normalise-input')
+const { supportsFileReader } = require('../../src/supports')
+const { Buffer } = require('buffer')
+const all = require('async-iterator-all')
+const pull = require('pull-stream')
+
+chai.use(dirtyChai)
+const expect = chai.expect
+
+const STRING = 'hello world'
+const BUFFER = Buffer.from(STRING)
+const ARRAY = Array.from(BUFFER)
+const TYPEDARRAY = Uint8Array.from(ARRAY)
+let BLOB
+
+if (supportsFileReader) {
+  BLOB = new global.Blob([
```

Comment:
```suggestion
  BLOB = new globalThis.Blob([
```
and require https://github.com/ipfs/js-ipfs-utils/blob/master/src/globalthis.js at the top